StripeCore/StripeCore/Source/API Bindings/STPAPIClient.swift

```diff
@@ -248,6 +248,18 @@ import UIKit
         client.userKeyLiveMode = userKeyLiveMode
         return client
     }
+
+    @_spi(STP) public class func paramsAddingClientAttributionMetadata(
+        _ params: [String: Any]
```

Comment:
Is there any good thing we could link to here if a reader wanted to know more about this metadata